diff --git a/dist/devtools/LGFFJRYV.js b/dist/devtools/LGFFJRYV.js
index d2625253d590a8df11e8fbd107fff14dcfacf10a..397c2f6b9a9c726276522d1f02b0fd4b10262c36 100644
--- a/dist/devtools/LGFFJRYV.js
+++ b/dist/devtools/LGFFJRYV.js
@@ -4562,17 +4562,18 @@ var SourceInspector = () => {
   });
   createEventListener(document, "click", (e) => {
     if (!highlightState.element) return;
+    const source = highlightState.dataSource;
     window.getSelection()?.removeAllRanges();
     e.preventDefault();
     e.stopPropagation();
     setDisabledAfterClick(true);
     if (settings().sourceAction === "copy-path") {
-      navigator.clipboard.writeText(highlightState.dataSource).catch(() => {
+      navigator.clipboard.writeText(source).catch(() => {
       });
       return;
     }
     const baseUrl = new URL(import.meta.env?.BASE_URL ?? "/", location.origin);
-    const url = new URL(`__tsd/open-source?source=${encodeURIComponent(highlightState.dataSource)}`, baseUrl);
+    const url = new URL(`__tsd/open-source?source=${encodeURIComponent(source)}`, baseUrl);
     fetch(url).catch(() => {
     });
   });
diff --git a/dist/devtools/MWM77JVE.js b/dist/devtools/MWM77JVE.js
index 336be09742a6523d89e08d0bfc98f1828b8aecc5..6223bf6214a7ee661a5c8090b303a2d2f2258968 100644
--- a/dist/devtools/MWM77JVE.js
+++ b/dist/devtools/MWM77JVE.js
@@ -3916,17 +3916,18 @@ var SourceInspector = () => {
   });
   createEventListener(document, "click", (e) => {
     if (!highlightState.element) return;
+    const source = highlightState.dataSource;
     window.getSelection()?.removeAllRanges();
     e.preventDefault();
     e.stopPropagation();
     setDisabledAfterClick(true);
     if (settings().sourceAction === "copy-path") {
-      navigator.clipboard.writeText(highlightState.dataSource).catch(() => {
+      navigator.clipboard.writeText(source).catch(() => {
       });
       return;
     }
     const baseUrl = new URL(import.meta.env?.BASE_URL ?? "/", location.origin);
-    const url = new URL(`__tsd/open-source?source=${encodeURIComponent(highlightState.dataSource)}`, baseUrl);
+    const url = new URL(`__tsd/open-source?source=${encodeURIComponent(source)}`, baseUrl);
     fetch(url).catch(() => {
     });
   });
