diff --git a/node_modules/starlight-site-graph/.bun-tag-aec78a799c19f4d0 b/.bun-tag-aec78a799c19f4d0
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/integration.ts b/integration.ts
index e538599644369dd2f2496759b754175b71832a5c..1734fd1e72200af2797931d748b09c6b328db215 100644
--- a/integration.ts
+++ b/integration.ts
@@ -50,8 +50,8 @@ export default defineIntegration({
 						outputPath = fileURLToPath(new URL(".vercel/output/static/", config.root));
 					} else if (config.adapter?.name === "@astrojs/cloudflare") {
 						outputPath = fileURLToPath(new URL(config.base?.replace(/^\//, ""), config.outDir));
-					} else if (config.adapter?.name === "@astrojs/node" && config.output === "server") {
-						outputPath = fileURLToPath(config.build.client!);
+					} else if (config.adapter && config.build.client) {
+						outputPath = fileURLToPath(config.build.client);
 					} else {
 						outputPath = fileURLToPath(config.outDir);
 					}
